gtest_parallel.py

```diff
@@ -757,14 +757,16 @@ def find_tests(binaries, additional_args, options, times):
     command = [test_binary] + additional_args
     if options.gtest_also_run_disabled_tests:
       command += ['--gtest_also_run_disabled_tests']
+    if options.size != '':
+      command += ['--size=' + options.size]
+      print('size filter: ON')
```

Comment:
Nit: should we print the size command?

Reply:
Ah. Whoops! I added that for debugging, meant to remove it.